src/widgets/Grabber.h

```diff
@@ -46,15 +46,13 @@ class GrabberEvent:public wxCommandEvent
    GrabberEvent(wxEventType type = wxEVT_NULL,
                 wxWindowID winid = 0,
                 const wxPoint& pt = wxDefaultPosition)
-   : wxCommandEvent(type, winid)
+   : wxCommandEvent(type, winid), mPos(pt)
```

Comment:
Just tried it with an even more complex "morecomplex" class:

```
class morecomplex
{
public:
   morecomplex()
   :  mParm(0),
      sss({2, 3, 4.f, 9.0})
   {
   }

   morecomplex(int parm)
   {
      mParm = parm;
   }
public:
   int mParm;
   char buffer[1234];
   struct
   {
      int a;
      int b;
      float f;
      double d;
   } sss;
};
```
And the results were exactly the same again.  Weird...wonder why the guys results were so much better...maybe something to do with templates?